IBPSA/Airflow/Multizone/DoorOpen.mo

```diff
@@ -24,8 +24,8 @@ protected
     "Polynomial coefficient for regularized implementation of flow resistance";
 
   parameter Real kVal=CD*AOpe*sqrt(2/rho_default) "Flow coefficient, k = V_flow/ dp^m";
-  parameter Real kT = CD*wOpe*sqrt(2/rho_default)
-    *(Modelica.Constants.g_n*rho_default*hOpe/2/Medium.T_default)^m *hOpe/2
+  parameter Real kT = CD*wOpe*hOpe/2*sqrt(2/rho_default)
+    *(Modelica.Constants.g_n*rho_default*(2*hOpe/9)/Medium.T_default)^m
```

Comment:
If I see this right, then the `hOpe/2` has just been moved around (no real change here), but the `1/2^m` has been changed to `(2/9)^m`. What is the explanation for this? If it is justified, can you please add it to the info section.

Reply:
The CONTAM user guide provides this information. They convert equation 69 to fit the powerlaw equation as defined by 
[1]W. G. Brown and K. R. Solvason, “Natural convection through rectangular openings in partitions—1 Vertical Partitions,” Int. J. Heat Mass Transfer, vol. 5, pp. 895–868, Mar. 1962.

This results in a stack effect of 4/9H between the two flow paths,  or twice 2/9H from the centerline.

![image](https://user-images.githubusercontent.com/46526840/105023526-5af53b00-5a4b-11eb-9b6d-897fa631e714.png)

Reply:
To be clear, the original implementation assumed a stack column height of hOpe between the two flow paths, which was an overestimation.